frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -46,6 +48,8 @@ export function removeLeakableResource(leakedResources: Set<string>, resource) {
 }
 
 export function createResource(resource) {
+  // ensure not return error even the source is existed by deleting it before adding it
```

Comment:
where is this necessary?

IMO we should have a specific method for this like `recreate` so the use case would be apparent by looking at the name.

We should not use this in places where it isn't necessary, so it would not hide bugs in the future

Reply:
I've seen some tests failed because of existed resource when I run tests locally, I was think it would be safe to make sure they're not returning error as they're used in setup/teardown.

I don't want to introduce `recreate` as they're not on the same purpose. I can revert this.

Reply:
they are not, but we could have both createResource and reCreateResource

ok, let's revert if you are not seeing this anymore. If you think it is necessary in some case, I would vote for having already mentioned functions and using it just in place where it is needed.

Reply:
What about add a check via `kubectl get`, if the resource is existing, do nothing. Then it don't throw error.

Reply:
Are you meaning to use `kubectl create` for createResource and use `kubectl apply` for applyResource?
I changed it to `kubectl apply` in createResource.

Reply:
yes, IMO it would be nicer to have both createResource and applyResource and use them accordingly

Reply:
I added applyResource, I will use it accordingly in future based on test experience.